source/taskmanager/src/main/resources/logback.xml

```diff
@@ -1,11 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <configuration>
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+      <pattern>%d [%t] %level %c{5} - %msg%n</pattern>
     </encoder>
   </appender>
  
-  <root level="info">
-    <appender-ref ref="STDOUT" />
+  <root>
+    <level value="WARN" />
+    <appender-ref ref="CONSOLE" />
   </root>
-</configuration>
\ No newline at end of file
+
+  <logger name="nl.aerius">
+    <level value="INFO" />
+  </logger>
+</configuration>
```

Comment:
The ref on line 11 doesn't match the name on line 3. They should be in sync.
I propose the following, this also includes short-hand notation to clean it a bit up.
Not so sure if the order is important when specifying the logger and root elements, but every example I could find seemed to do it this way.
Might simply be so you specify the specific things first and end with the default.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<configuration>
  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
    <encoder>
      <pattern>%d [%t] %level %c{5} - %msg%n</pattern>
    </encoder>
  </appender>

  <logger name="nl.aerius" level="info" />
 
  <root level="warn">
    <appender-ref ref="STDOUT" />
  </root>
</configuration>
```